saray/weather-card/stylesheet.css

```diff
@@ -0,0 +1,45 @@
+.weather-card--container {
+    background-color: #FBED78;
+    border-radius: 4px;
+    width: 300px;
+    padding: 15px 20px;
+    font-family: 'Montserrat', sans-serif;
+    line-height: 1.1em;
+}
```

Comment:
You can define the color in the element container. This is an Inherited property, you don’t will need to specify in each element.